pkgs/std/memfile.go

```diff
@@ -30,6 +32,14 @@ func (mempkg *MemPackage) GetFile(name string) *MemFile {
 	return nil
 }
 
+func (mempkg *MemPackage) GetFileBodies() MemFileBodies {
```

Comment:
```suggestion
func (mempkg *MemPackage) GetFileBodies() map[string]string {
```

If you anticipate needing more helpers in the future, you can disregard this suggestion. However, if not, I believe this approach will be more discoverable and easier to upgrade later.